corehq/apps/accounting/utils/invoicing.py

```diff
@@ -40,38 +45,48 @@ def get_oldest_overdue_invoice_over_threshold(today, domain):
 
 
 def get_domains_with_subscription_invoices_overdue(today):
-    for domain in set(_get_unpaid_saas_invoices_in_downgrade_daterange(today).values_list(
+    invoices = _get_unpaid_saas_invoices_in_downgrade_daterange(today)
+    return _get_domains_over_threshold(invoices, today, get_oldest_overdue_invoice_over_threshold)
+
+
+def _get_domains_over_threshold(invoices, today, get_oldest_invoice_fn):
```

Comment:
similarly confusing name to the `_get_accounts_over_threshold` based on what it yields.

maybe for this function, `_get_overdue_invoice_criteria_for_domains`?

it might be helpful to add doc strings about what the expected value is and what the params are, and what the function's purpose is